src/dune_engine/action_builder.mli

```diff
@@ -117,6 +117,10 @@ val path_set : Path.Set.t -> unit t
     dependencies of the action produced by the action builder. *)
 val paths_matching : loc:Loc.t -> File_selector.t -> Path.Set.t t
 
+(** Like [paths_matching], but don't return the resulting set. The action
+    dependency is still registered. *)
+val paths_matching_ : loc:Loc.t -> File_selector.t -> unit t
```

Comment:
What do you of think of `paths_matching_unit`? That's the suffix we use for `dyn_paths_unit`.

Reply:
It's a bit odd to read though, feels like "paths that are matching unit".

Reply:
Either name seems OK to me. I changed it to `paths_matching_unit`.